packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html

```diff
@@ -130,6 +130,14 @@ <h2>
             data-editable="!$ctrl.isSubmit"
             data-on-submit="$ctrl.forceSubmitContainer(form)"
         >
+            <oui-message type="info" dismissable="true">
+                {{::
+                'pci_projects_project_training_jobs_submit_select_data_tips_1' |
+                translate}}<br />
```

Comment:
If the only purpose is to have a return to line, using an element with `display:block` should be done instead

Reply:
it is the only purpose, which element should I use ?

Reply:
```suggestion
                <span class="d-block" data-translate="pci_projects_project_training_jobs_submit_select_data_tips_1"></span>
```
or
```suggestion
                <p class="d-block" data-translate="pci_projects_project_training_jobs_submit_select_data_tips_1"></p>
```

Reply:
Fixed